lms/djangoapps/certificates/views/webview.py

```diff
@@ -97,7 +97,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=_(user_certificate.modified_date.strftime("%B")),
```

Comment:
@amangano-edx Thank you. We will try to rebase this one with `master` branch and if the update is ok, we will make a new commit with the changes on this same the PR. As the PR is from April, maybe the rebase is not completely ok, in that case we will create a new PR referencing this one.